StickyTabBarViewController/Classes/ExpandableViewController.swift

```diff
@@ -14,7 +14,7 @@ public protocol Expandable: UIViewController {
 }
 
 public extension Expandable {
-    var container: StickyViewControllerSupporting? {
+    weak var container: StickyViewControllerSupporting? {
```

Comment:
I made it `weak` to make sure there won't be any retain cycle between tab bar and child VC, but when I try it without making it weak, I couldn't see any cycles either. Maybe this was unnecessary here.

Reply:
hmm good finding. I'm wondering if it's because it's returning the `tabBarController` that every UIController has but just casting it to `StickyViewControllerSupporting`

So maybe internally this is already handled by UIKit. 

I'd leave like this if it's not hurting anything. I my mind it makes it explicit, even if there's no retain cycle.

Reply:
👍